src/integration/integration_metricscollector_eventgenerator_scalingengine_test.go

```diff
@@ -0,0 +1,34 @@
+package integration_test
+
+import (
+	. "integration"
+
+	"autoscaler/cf"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	// . "github.com/onsi/gomega"
+)
+
+var _ = Describe("Integration_Metricscollector_Eventgenerator_Scalingengine", func() {
+
+	BeforeEach(func() {
+		startFakeCCNOAAUAA()
+		metricsCollectorConfPath = prepareMetricsCollectorConfig(dbUrl, components.Ports[MetricsCollector], fakeCCNOAAUAA.URL(), cf.GrantTypePassword)
+		eventGeneratorConfPath = prepareEventGeneratorConfig(dbUrl, components.Ports[EventGenerator], fmt.Sprintf("http://127.0.0.1:%d", components.Ports[MetricsCollector]), fmt.Sprintf("http://127.0.0.1:%d", components.Ports[ScalingEngine]))
+		scalingEngineConfPath = prepareScalingEngineConfig(dbUrl, components.Ports[ScalingEngine], fakeCCNOAAUAA.URL(), cf.GrantTypePassword)
+		startMetricsCollector()
+		startEventGenerator()
+		startScalingEngine()
+	})
+
+	AfterEach(func() {
+		stopAll()
+	})
+	Describe("", func() {
```

Comment:
For this pull request I just put the start components functions here. So all the integration test can use these functions.